capstone/src/main/webapp/script.js

```diff
@@ -316,32 +305,48 @@ async function averagePrecinctSentiment(policePrecinct) {
     }
   }
   const sentimentAverage = sentimentCount/list.length;
-  return getSentimentColor(sentimentAverage);
+  return getSentimentColor(sentimentAverage, policePrecinct);
 }
 
-function getSentimentColor(averageFeelings) {
+function getSentimentColor(averageFeelings, policePrecinct) {
+    let precinctColor = '#228B22';
   if (Math.round(averageFeelings) == 5) {
-    return '#165B33';
+    precinctColor = '#165B33';
   } else if (Math.round(averageFeelings) == 4) {
-    return '#146B3A';
+    precinctColor = '#146B3A';
   } else if (Math.round(averageFeelings) == 3) {
-    return '#F8B229';
+    precinctColor = '#F8B229';
   } else if (Math.round(averageFeelings) == 2) {
-    return '#EA4630';
+    precinctColor = '#EA4630';
   } else {
-    return '#BB2528';
+    precinctColor = '#BB2528';
   }
+   const precinctLayer = mapAndSelection.map;
+
+    if(policePrecinct!='Richmond'&& policePrecinct!='Southern'){
+        precinctLayer.overrideStyle(precinctLayer.getFeatureById(
+            searchPrecinctsByStation(policePrecinct)),
+        {fillColor: precinctColor, fillOpacity: 0.9});
+    }
 }
 
 const mapAndSelection = {};
 /* eslint-disable no-unused-vars */
 function onlyOne(checkbox) {
```

Comment:
Can you add documentation to explain what this function does? It would be good to add documentation to the other functions as well

Reply:
I've added documentation for every function in script.js